app/core/src/main/java/com/fsck/k9/message/TextBodyBuilder.java

```diff
@@ -12,6 +11,10 @@
 
 
 class TextBodyBuilder {
+    public static final String HTML_AND_BODY_START =
+            "<!DOCTYPE html PUBLIC \"-//W3C//DTD HTML 4.0 Transitional//EN\"><html><body>";
```

Comment:
Let's go with HTML 5 and make this `<!DOCTYPE html><html><body>`

Reply:
Done